examples/experimental-9p.yaml

```diff
@@ -0,0 +1,22 @@
+# This example requires Lima v0.7.0 or later.
```

Comment:
```suggestion
# This example requires Lima v0.10.0 or later.
# On macOS hosts, QEMU needs to be v7.0.0 or later. Homebrew's QEMU v6.2.0_1 can be used too.
# This example is planned to be merged to default.yaml in Lima v1.0 (ETA: 2022 Q2).
```

Reply:
Also, could you run this in CI?

Reply:
> Also, could you run this in CI?

Do you have hints ? Copy/paste something in `.github` ?

Reply:
https://github.com/lima-vm/lima/blob/ff72c3ae03c4d657cc6c897850c6c89af88bdd3c/.github/workflows/test.yml#L119-L139